.github/workflows/bump-main-loculus-version-to-latest.yaml

```diff
@@ -30,7 +31,11 @@ jobs:
         id: changelog
         run: |
           cd loculus_repo
-          CHANGELOG=$(git log --pretty=format:"- %s" ${{ steps.get_current_version.outputs.CURRENT_VERSION }}..${{ steps.get_loculus_commit.outputs.LATEST_SHA }})
+          CHANGELOG=$( \
+            git log --pretty=format:"- %s" \
```

Comment:
This is cool but I actually find it quite hard to read due to the duplication ? Maybe we could just have the links?

Reply:
Or could explicitly give the link text with markdown

Reply:
Sorry I replied to this in the other chat